boltdesignsystem/bolt
# DS-816 Update twig templates that use contentItems

## Jira

https://pegadigitalit.atlassian.net/browse/DS-816

## Summary

The `contentItems` was replaced with `items` or `content` in a bunch of twig templates and the usage was updated in the PL.

## Details

I checkout the branch from the [DS-790](https://github.com/boltdesignsystem/bolt/pull/2510) .

The twig templates that were updated:

:small_blue_diamond: `_band--collection.twig` and its usage in:
- `Experiments`
- `Archive`
--events
--homepage
--press and media
--product-pages

:point_right: _`contentItems` was replaced with `content` because in a **Band** `items` prop is deprecated and it threw an error when used `items`_


:small_blue_diamond: `_mission-landing.twig` and its usage in: 
- `Archive`
--Academy


:small_blue_diamond: `_dashboard-band.twig` and its usage in: 
- `Components`
-- Archive Carousel
- `Archive`
--Academy

## How to test

Pull the branch. Check if there are no regressions caused by replacing `contentItems` prop.
